app/assets/stylesheets/application.css.scss

```diff
@@ -143,6 +143,8 @@ h1.title {
 }
 h2 {
   font-size: 2.215rem;
+  padding-top: 15px;
```

Comment:
Can I ask about this? I'm a little wary of editing the global style for h2 unless we have a good reason -- can we get away without tweaking this?